src/Symfony/Component/HttpKernel/Log/Logger.php

```diff
@@ -39,6 +40,9 @@ class Logger extends AbstractLogger
     /** @var resource|null */
     private $handle;
 
+    /** @var Recorder|null */
+    private $recorder;
```

Comment:
```suggestion
    private ?Recorder $recorder = null;
```

Reply:
Should I update other properties as well?

Reply:
that's already done ;)

Reply:
Damn I thought you could typehint `resource` :disappointed_relieved: